### PR TITLE
DB-10265 remove outdated / insensitive terminology

### PR DESCRIPTION
**Pull request by martinrupp**

### CODE REVIEW

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by arnaud-splice — APPROVED** [review]

**Review by ipraznik-splice — APPROVED** [review]

**Review by jyuanca — APPROVED** [review]

**Review by carolp-503 — APPROVED** [review]
+1

File: splice_machine/src/test/java/com/splicemachine/triggers/Trigger_Exec_Stored_Proc_IT.java [hunk 1]
```diff
@@ -416,20 +416,20 @@ public void testStatementTriggerSysStoredProc() throws Exception {
     @Test @Ignore("DB-3424: Executing trigger action insert thru a stored proc (disallowed directly) causes infinite recursion.")
     public void testExecSQLRecursion() throws Exception {
 
-        methodWatcher.executeUpdate("insert into t_master values (13, 'grrr', 'I''m a pirate!', CURRENT_TIMESTAMP)");
-        methodWatcher.executeUpdate("insert into t_slave values (99, 'trigger01', CURRENT_TIMESTAMP)");
-        methodWatcher.executeUpdate("insert into t_slave2 values (22, 'trigger02', CURRENT_TIMESTAMP)");
+        methodWatcher.executeUpdate("insert into t_first values (13, 'grrr', 'I''m a pirate!', CURRENT_TIMESTAMP)");
+        methodWatcher.executeUpdate("insert into t_second values (99, 'trigger01', CURRENT_TIMESTAMP)");
+        methodWatcher.executeUpdate("insert into t_second2 values (22, 'trigger02', CURRENT_TIMESTAMP)");
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/triggers/Trigger_Exec_Stored_Proc_IT.java:421]: The "second2" naming is a bit confusing, I'd like more primary/secondary/secondary2, source/sink/sink2, source/target/target2 ...

**Review by dgomezferro — APPROVED** [review]

**Pull request merged**